.github/workflows/test.yml

```diff
@@ -23,9 +24,6 @@ jobs:
       - name: Install requirements
         run: pip install -r seed/requirements.txt
 
-      - name: Serialize seed
-        run: python seed/serialize.py seed/seed.json
-
       - name: Generate testing_fieldtrials.json (HEAD)
```

Comment:
let's rename this workflow file to be something like `test-testing-fieldtrials-gen.yml`, and also rename `name: Test Python Generator` to a better suitable name.

Reply:
done.